drivers/hmis/app/graphql/types/hmis_schema/ce_referral_step.rb

```diff
@@ -8,37 +8,72 @@
 
 module Types
   class HmisSchema::CeReferralStep < Types::BaseObject
-    # object is a Hmis::WorkflowExecution::Step
+    # object is an OpenStruct with:
+    # {
+    #   step: Hmis::WorkflowExecution::Step,
+    #   referral: Hmis::Ce::Referral,
+    # }
+    # Why: The Step schema object should resolve this step's potential Participants (ReferralParticipants on the Referral).
+    # However, the Step model does not have a direct relationship to Referral, to avoid a duplicative bidirectional relationship.
 
     field :id, ID, null: false, description: 'unique identifier for this step based on node and instance'
     field :step_id, ID, null: true, method: :id, description: 'the DB identifier of this step, if it is persisted'
     field :form_definition, Types::Forms::FormDefinition, null: false
     field :name, String, null: false
     field :status, HmisSchema::Enums::CeReferralStepStatus, null: false
-    field :swimlane, String, null: false
     field :submitted_values, JsonObject, null: true
-    delegate :name, to: :workflow_node
+    field :swimlane, HmisSchema::CeReferralSwimlane, null: true, description: 'Swimlane for this step, which holds information about potential step participants'
+    field :assignees, [Application::User], null: true, description: 'User(s) currently assigned to this step'
```

Comment:
again since this is an array it can be `null: false`, unless we have reason to differentiate between null and `[]`